models/convert/cadence.go

```diff
@@ -26,8 +26,8 @@ import (
 
 func ParseCadenceArgument(param string) (cadence.Value, error) {
 
-	// Cadence values should be provided in the form of Type(Value), so that we
-	// can unambiguously determine the type.
+	// Cadence values should be provided in the form of Type(Value), so that
+	// their type can be determined unambiguously.
```

Comment:
I really don't see the point of these changes if these comments are clearly part of the code flow and meant to help people understand the code. They are not part of the documentation in any way.